src/test/java/com/karankumar/bookproject/ui/book/BookFormTests.java

```diff
@@ -106,29 +117,36 @@ private Book createBook(PredefinedShelfService predefinedShelfService) {
         book.setDateStartedReading(dateStarted);
         book.setDateFinishedReading(dateFinished);
         book.setRating(ratingVal);
+        if (isInSeries) {
+            book.setSeriesPosition(seriesPosition);
+        }
 
         return book;
     }
 
     private int generateRandomNumberOfPages() {
         return ThreadLocalRandom.current().nextInt(300, (2000 + 1));
     }
+    private int generateRandomSeriesPosition() {
+        return ThreadLocalRandom.current().nextInt(1, (10 + 1));
+    }
 
     /**
      * Tests whether the form fields are correctly populated
      */
     @Test
     public void formFieldsPopulated() {
-        Assertions.assertEquals(bookTitle, bookForm.bookTitle.getValue());
-        Assertions.assertEquals(firstName, bookForm.authorFirstName.getValue());
-        Assertions.assertEquals(lastName, bookForm.authorLastName.getValue());
-        Assertions.assertEquals(readShelf.getPredefinedShelfName(), bookForm.shelf.getValue());
-        Assertions.assertEquals(genre, bookForm.bookGenre.getValue());
-        Assertions.assertEquals(pagesRead, bookForm.pagesRead.getValue());
-        Assertions.assertEquals(numberOfPages, bookForm.numberOfPages.getValue());
-        Assertions.assertEquals(dateStarted, bookForm.dateStartedReading.getValue());
-        Assertions.assertEquals(dateFinished, bookForm.dateFinishedReading.getValue());
-        Assertions.assertEquals(rating, bookForm.rating.getValue());
+        assertEquals(bookTitle, bookForm.bookTitle.getValue());
```

Comment:
Could you change the asserts back to use the qualified class name (`Assertions.assertEquals`), as with your other PR?